frontend/setupChelonia.js

```diff
@@ -141,8 +179,15 @@ const setupChelonia = async (): Promise<*> => {
       }
     },
     hooks: {
-      syncContractError: (e: Error, contractID: string) => {
-        if (['ChelErrorUnrecoverable', 'ChelErrorForkedChain'].includes(e?.name)) {
+      syncContractError: (e, contractID) => {
+        if (!e) return
+        if (e.name === 'ChelErrorUnexpectedHttpResponseCode' && e.message.startsWith('410:')) {
+          console.info('[syncContractError] Contract ID ' + contractID + ' has been deleted')
+          handleDeletedContract(contractID).catch(e => {
+            console.error('Error handling contract deletion', e)
```

Comment:
To help us debug problems, console error messages should be unique. This `'Error handling contract deletion'` message appears identically in 3 different places. Could you please make each version of it unique so that when reviewing logs we can identify where it came from?